src/Acuminator/Acuminator.Analyzers/StaticAnalysis/SavingChanges/SaveOperationHelper.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Acuminator.Utilities.Common;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Acuminator.Analyzers.StaticAnalysis.SavingChanges
+{
+	internal static class SaveOperationHelper
+	{
+		private const string GraphPersistMethodName = "Persist"; // PXGraph.Persist
+		private const string PressSaveMethodName = "PressSave"; // PXActionCollection.PressSave
+		private const string PressMethodName = "Press"; // PXAction.Press
+		private const string CachePersistMethodName = "Persist"; // PXCache.Persist
+		private const string CachePersistInserted = "PersistInserted"; // PXCache.PersistInserted
+		private const string CachePersistUpdated = "PersistUpdated"; // PXCache.PersistUpdated
+		private const string CachePersistDeleted = "PersistDeleted"; // PXCache.PersistDeleted
+
+		public static SaveOperationKind GetSaveOperationKind(IMethodSymbol symbol, InvocationExpressionSyntax syntaxNode, 
+			SemanticModel semanticModel, PXContext pxContext)
+		{
+			symbol.ThrowOnNull(nameof (symbol));
+			syntaxNode.ThrowOnNull(nameof (syntaxNode));
+			semanticModel.ThrowOnNull(nameof (semanticModel));
+			pxContext.ThrowOnNull(nameof (pxContext));
+
+			if (symbol.ContainingType != null)
+			{
+				// PXGraph.Actions.PressSave or PXSave.Press
+				if (String.Equals(symbol.Name, PressSaveMethodName, StringComparison.Ordinal)
```

Comment:
@dropsonic are String.Equals necessary here? We are not working with UI strings and could rely on standard `==` which is ordinal by default and is shorter

Reply:
@dropsonic also isn't a swith with pattern matching a little more readable?

Reply:
@SENya1990 you're completely right, I'll fix it